ui/apps/platform/src/Containers/Clusters/InitBundles/InitBundleWizardStep1.tsx

```diff
@@ -0,0 +1,122 @@
+            <Form>
+                <FormLabelGroup
+                    fieldId="name"
+                    label="Name"
+                    isRequired
+                    errors={errors}
+                    touched={touched}
+                >
+                    <TextInput
+                        type="text"
+                        id="name"
+                        name="name"
+                        isRequired
+                        value={values.name}
+                        onBlur={handleBlur}
+                        onChange={onChange}
+                    />
+                </FormLabelGroup>
+                <Alert
+                    variant="info"
+                    isInline
+                    title="You can use one bundle for multiple clusters on the same platform with the same installation method."
+                    component="p"
+                />
+                <FormGroup fieldId="platform" label="Platform of secured clusters" isRequired>
+                    <Flex
+                        direction={{ default: 'column' }}
+                        spaceItems={{ default: 'spaceItemsSm' }}
+                    >
+                        {Object.entries(platformOptions).map(([platformKey, platformLabel]) => (
+                            <Radio
+                                key={platformKey}
+                                name={platformKey}
+                                value={platformKey}
+                                onChange={() => {
+                                    setFieldValue('platform', platformKey);
+                                    if (platformKey !== 'OpenShift') {
+                                        setFieldValue('installation', 'Helm');
+                                    }
+                                }}
+                                label={platformLabel}
+                                id={platformKey}
+                                isChecked={values.platform === platformKey}
+                            />
+                        ))}
+                    </Flex>
+                </FormGroup>
+                <FormGroup
+                    fieldId="installation"
+                    label="Installation method for secured cluster services"
+                    isRequired
+                >
+                    <Select
+                        variant="single"
+                        toggleAriaLabel="Installation method menu toggle"
+                        aria-label="Select an installation method"
+                        isDisabled={values.platform !== 'OpenShift'}
+                        onToggle={installationToggle.onToggle}
+                        onSelect={(_event, value) => {
+                            setFieldValue('installation', value);
+                        }}
+                        selections={values.installation}
+                        isOpen={installationToggle.isOpen}
+                        // className="pf-u-flex-basis-0"
```

Comment:
is this supposed to be commented out here?

Reply:
Good eyes! There are 4 results in 3 files, so I need to investigate what is the purpose and is it needed here.